contents.json

```diff
@@ -5791,6 +5791,11 @@
       "category": "dependency-injection",
       "description": "Dependency Injection Framework for Swift, inspired by KOIN.",
       "homepage": "https://github.com/Liftric/DIKit"
+    }, {
+      "title": "MockSwift",
+      "category": "mock",
+      "description": "Mock Framework for Swift that use the power of property wrappers.",
```

Comment:
@leoture could you change the description to "Mock Framework that uses the power of property wrappers."?